debian/control

```diff
@@ -6,22 +5,30 @@ Maintainer: Go Compiler Team <pkg-golang-devel@lists.alioth.debian.org>
 Uploaders: Michael Stapelberg <stapelberg@debian.org>,
            Paul Tagliamonte <paultag@debian.org>,
            Tianon Gravi <admwiggin@gmail.com>
-Vcs-Browser: http://anonscm.debian.org/gitweb/?p=pkg-golang/golang.git
+Vcs-Browser: https://anonscm.debian.org/cgit/pkg-golang/golang.git
 Vcs-Git: git://anonscm.debian.org/pkg-golang/golang.git
-Build-Depends: debhelper (>= 7.4.10), bison, ed, mawk | awk, perl, netbase
+Build-Depends: bison,
+               debhelper (>= 7.4.10),
+               ed,
+               golang (>= 1:1.4~) | gccgo-5,
+               mawk | awk,
+               netbase,
+               perl
 Build-Depends-Indep: po-debconf
 Standards-Version: 3.9.6
-Homepage: http://golang.org/
+Homepage: https://golang.org
 
 Package: golang-go
-Architecture: i386 amd64 armel armhf
-Depends: ${shlibs:Depends}, ${misc:Depends}, ${perl:Depends},
-	 golang-src (>= ${source:Version}),
-	 golang-go-linux-${go:Hostarch} (= ${source:Version})
-Replaces: golang-weekly-go, golang-weekly-tools, golang-tools
-Conflicts: golang-weekly-go, golang-weekly-tools, golang-tools
-Provides: go-compiler, golang-weekly-go, golang-weekly-tools, golang-tools
-Suggests: bzr, git, golang-go.tools, mercurial, subversion
+Architecture: amd64 arm64 armel armhf i386 ppc64 ppc64el
```

Comment:
The idea is that we let the things we depend on dictate which platforms we're supported on as far as I know -- I'll be honest that I don't fully understand why one would be preferred over the other, and am just basing this off how @paultag's had `src:docker.io` setup since very early on in my involvement. :+1: